README.md

```diff
@@ -143,6 +148,11 @@ causes "lag"/pauses in the UI. To workaround this you can disable Prometheus met
 --ResourceUseDisplay.enable_prometheus_metrics=False
 ```
 
+### Show or Hide information from sidebar (tachometer)
+If you want to show or hide information from the Kernel Usage sidebar
```

Comment:
The same suggestion on wording here. Did it need to be reported twice or could one section link to the other?

Reply:
yes, I think once should be enough.